covariancematrix.go

```diff
@@ -13,26 +13,23 @@ import (
 
 // CovarianceMatrix calculates a covariance matrix (also known as a
 // variance-covariance matrix) from a matrix of data, using a two-pass
-// algorithm. The matrix returned will be symmetric and square.
+// algorithm.
 //
 // The weights wts should have the length equal to the number of rows in
 // input data matrix x. If c is nil, then a new matrix with appropriate size will
-// be constructed.  If c is not nil, it should be a square matrix with the same
-// number of columns as the input data matrix x, and it will be used as the receiver
-// for the covariance data.  Weights cannot be negative.
-func CovarianceMatrix(cov *mat64.Dense, x mat64.Matrix, wts []float64) *mat64.Dense {
+// be constructed. If c is not nil, it should have a number of columns as the
+// input data matrix x, and it will be used as the receiver for the covariance
```

Comment:
Right. Sorry.
> On Sep 29, 2015, at 12:26 AM, Dan Kortschak <notifications@github.com> wrote:
> 
> In covariancematrix.go <https://github.com/gonum/stat/pull/55#discussion_r40638125>:
> 
> >  //
> >  // The weights wts should have the length equal to the number of rows in
> >  // input data matrix x. If c is nil, then a new matrix with appropriate size will
> > -// be constructed.  If c is not nil, it should be a square matrix with the same
> > -// number of columns as the input data matrix x, and it will be used as the receiver
> > -// for the covariance data.  Weights cannot be negative.
> > -func CovarianceMatrix(cov *mat64.Dense, x mat64.Matrix, wts []float64) *mat64.Dense {
> > +// be constructed. If c is not nil, it should have a number of columns as the
> > +// input data matrix x, and it will be used as the receiver for the covariance
> s/receiver/destination/
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/gonum/stat/pull/55/files#r40638125>.
>